frontend/src/locales/en-CH-scout.json

```diff
@@ -1,4 +1,10 @@
 {
+  "contentNode": {
+    "storycontext": {
+      "add": "Add story",
+      "name": "Story"
```

Comment:
In this case, this might be obsolete. In German we have the difference "Geschichte" (normal) vs. "Roter Faden" (Pfadislang).

So lange wir nicht wissen, dass im Englischen Pfadislang auch etwas anderes als "Story" verwendet wird, lassen wir das wohl besser weg.